.travis.yml

```diff
@@ -1,6 +1,25 @@
-language: haskell
-ghc:
-  - 7.4
-  - 7.6
-  - 7.8
-script: "cabal build && cabal test && cabal sdist"
\ No newline at end of file
+env:
+  - GHC_VERSION=7.4.2  CABAL_VERSION=1.18
+  - GHC_VERSION=7.6.3  CABAL_VERSION=1.20
+  - GHC_VERSION=7.8.4  CABAL_VERSION=1.20
+  - GHC_VERSION=7.10.1 CABAL_VERSION=1.22
+
+matrix:
+  allow_failures:
+    - env: GHC_VERSION=7.10.1 CABAL_VERSION=1.22        
```

Comment:
@jsl I add this line because the build is broken for GHC 7.10.1 but seems related to `keyword-args` and not `openssh-github-keys` itself

```
cabal: Could not resolve dependencies:
trying: openssh-github-keys-0.1.0.0 (user goal)
trying: base-4.8.0.0/installed-1b6... (dependency of
openssh-github-keys-0.1.0.0)
next goal: keyword-args (dependency of openssh-github-keys-0.1.0.0)
rejecting: keyword-args-0.2.0.1, 0.1.0.5, 0.1.0.4, 0.1.0.3, 0.1.0.2, 0.1.0.1,
0.1.0.0 (conflict: base==4.8.0.0/installed-1b6..., keyword-args => base>=4.5
&& <4.8)
```

https://travis-ci.org/stackbuilders/openssh-github-keys/jobs/59728240